### ISSUE TITLE
Include source maps in ESM build?

### ISSUE THREAD

**Issue by jbaiter** [IIIF-Commons/vault-helpers#8]
I'm currently trying to debug an issue involving the thumbnail helper, and it'd be really helpful if source maps were included in the minified builds. This could also help with e.g. Sentry error reporting.
Alternatively, maybe there could also be an unminified build?
This would also apply to the `vault` package

**Comment by stephenwf** [comment]
I had a crack at adding in source maps (Typescript -> esbuild -> rollup) and something in that chain broke it:
```json
{"version":3,"file":null,"sources":[null],"sourcesContent":[null],"names":[],"mappings":" ... "}
```

I think I'll look to migrate the normal rollup config to `vite` - which uses rollup internally, but should ease configuration.

**Comment by stephenwf** [comment]
Should be closed by #10 which contains a switch to `vite`
